### DISCUSSION TITLE
FATAL: remaining connection slots are reserved for non-replication superuser connections

### DISCUSSION THREAD

**Discussion by MerlinB** [nextcloud/all-in-one#965] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Set up Nextcloud AIO
2. Import calendar with 400+ entries

### Expected behavior 
The calendar gets imported

### Actual behavior 
After a few seconds several errors appear and the site reloads. The calendar is only partly imported.
The same thing happens when trying to import contacts and possibly when uploading many files as well.

It seems that somehow new database connections are endlessly created until the system fails.

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
v1.5.0

#### Current channel 
latest

#### Other valuable info 
![image](https://user-images.githubusercontent.com/7633719/180197482-13749ea8-df9f-4d23-af53-97a94e444411.png)

Here is the relevant section of nextcloud.log:
[nextcloud (1).log](https://github.com/nextcloud/all-in-one/files/9158737/nextcloud.1.log)

There are no errors in the postgres or nextcloud docker container logs.

Note: I have followed the migration guide to migrate my old files.

**Comment by szaimen** [comment]
So there is really nothing in the postgres logs? It sounds like there should be something as the max connections seems to be reached...

Either way I fear there is no good way to fix this on our side. The import should rather not open as many databaae connections... Please report this to the calendar app!

However I'll move this to discussions to see if we should raise the max connections value.

**Reply by szaimen** [reply]
> However I'll move this to discussions to see if we should raise the max connections value.

Seems like there is indeed one: https://github.com/nextcloud/all-in-one/issues/966

**Comment by szaimen** [comment, marked as answer]
Lets handle this further in https://github.com/nextcloud/all-in-one/issues/966
